snooty.toml

```diff
@@ -7,7 +7,9 @@ intersphinx = [ "https://www.mongodb.com/docs/manual/objects.inv",
 
 sharedinclude_root = "https://raw.githubusercontent.com/10gen/docs-shared/main/"
 
-toc_landing_pages = []
+toc_landing_pages = [
+    "/bson.txt/"
```

Comment:
Suggestion:
Although this appears to work, the notation we usually use omits the .txt and trailing slash. E.g.
"/bson"